crates/astria-sequencer/src/utils.rs

```diff
@@ -13,17 +13,6 @@ use tendermint::abci::{
     EventAttributeIndexExt as _,
 };
 
-pub(crate) struct Hex<'a>(pub(crate) &'a [u8]);
```

Comment:
This was put in to hex-components of state keys be human readable.

Reply:
I think this is fine to remove, as we've never actually needed the storage keys to be human-readable

Reply:
We were using the `hex` crate for decoding, so I didn't see a problem in using it for the encoding too.  A little less code, and also a bit more performant than our own.

I should probably have called this out in the PR description though - sorry!

Reply:
The hex crate always allocates though. So for case `Hex(your_bytes).to_string()` I would prefer to use `hex` directly. But using `hex::encode` when constructing state keys means you always do an extra String allocation before constructing your final string.

I'd rather leave this here and optimize it.

Reply:
As discussed elsewhere, I'll revert this and we can handle it in a follow-up PR.  Ticketed at #1554.